mmcv/transforms/processing.py

```diff
@@ -1027,19 +1027,21 @@ class RandomFlip(BaseTransform):
     - flip_direction
 
     Args:
-         prob (float | list[float], optional): The flipping probability.
-             Defaults to None.
-         direction(str | list[str]): The flipping direction. Options
-             If input is a list, the length must equal ``prob``. Each
-             element in ``prob`` indicates the flip probability of
-             corresponding direction. Defaults to 'horizontal'.
+        prob (float | list[float], optional): The flipping probability.
+            Defaults to None.
+        direction(str | list[str]): The flipping direction. Options
+            If input is a list, the length must equal ``prob``. Each
+            element in ``prob`` indicates the flip probability of
+            corresponding direction. Defaults to 'horizontal'.
+        swap_label_pairs (list, optional): The label pair need to be swapped
+            for ground truth, like 'left arm' and 'right arm' need to be
+            swapped after horizontal flipping.
```

Comment:
Does the `swap_label_paris` only affect seg maps? How about key points?

Reply:
And better to provide the detailed usage of the argument. How to specify the `left arm` and `right arm` pair?